src/main/java/com/commercetools/sync/inventories/SkuKeyTuple.java

```diff
@@ -0,0 +1,117 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+/**
+ * Helper class that holds {@code sku} of {@link io.sphere.sdk.inventory.InventoryEntry} and the {@code key} of the
+ * {@link io.sphere.sdk.channels.Channel} it belongs to. Presence of sku and key in created instance is similar as for
+ * inventory entry resource: supply channel key can be {@code null} value since it is optional in inventory entry,
+ * but sku has to be not null neither not empty.
+ * Can be used as key in Maps.
+ */
+final class SkuKeyTuple {
```

Comment:
SkuKey seems too vague. As it is strictly for inventories I'd name it SkuChannelKeyTuple so that it is
immediately clear what key is held here